src/app-feature/profile-feature/components/password-change-form/password-change-form.component.html

```diff
@@ -19,15 +19,15 @@ <h2>{{ 'Change password' | translate }}</h2>
     <label class="app-form__element--block">
       <span class="app-form__element-label">{{ 'New Password:' | translate }}</span>
       <span class="app-form__element-input">
-        <input type="password" class="app-form__text-input" formControlName="newPassword">
+        <input type="password" class="app-form__text-input" formControlName="newPassword" name="newPassword">
         <app-form-control-errors namespace="passwordChangeForm.errors" formControlPath="newPasswordsGroup.newPassword"></app-form-control-errors>
       </span>
     </label>
 
     <label class="app-form__element--block">
       <span class="app-form__element-label">{{ 'Confirm Password:' | translate }}</span>
       <span class="app-form__element-input">
-        <input type="password" class="app-form__text-input" formControlName="confirmPassword">
+        <input type="password" class="app-form__text-input" formControlName="confirmPassword" name="confirmPassword">
```

Comment:
We'll have to create custom Protractor locators that find components by looking for `formControlName`, I want to avoid using CSS selectors. Alternatively, we should probably look into testing pages by hooking into accessibility layer (using ARIA roles and what not).

Reply:
It is one option, although when you usually rename the CSS class it most likely because of the element changes it's meaning, implicitly also the locator might need to change to keep in sync. Having this, it could be double work, let's discuss this in our meeting.